<?xml version="1.0" encoding="UTF-8"?>
<launch>
	<arg name="image_path" default="$(find tb_bringup)/heightimages/d2.png"/>
	<arg name="radius_surroundcloud" default="25.0"/>
	<arg name="image_elevation_max"  default="50.0"/>
	<arg name="limit_dhdng" default="1.5"/>
	<arg name="limit_dincl"  default="1.0"/>
	<arg name="sensor_range"  default="50.0"/>

	<node name="tb_img2fullcloud_node"  pkg="tb_simulate" type="tb_img2fullcloud_node.py" output="screen" respawn="true">
		<param name="image_path" 											 type="string" value="$(arg image_path)" />
		<param name="normalized_image_zmax"	 					 type="double" value="$(arg image_elevation_max)" />
		<param name="image_is_normalized"	 				 		 type="bool"   value="true" />
	</node>

</launch>
